### PR TITLE
make bdr ddl replicate GA en

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/18345
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-bidirectional-replication.md [hunk 1]
```diff
@@ -89,14 +89,12 @@ Non-replicable DDLs include:
...
+- `PRIMARY`: You can execute replicable DDLs, but not non-replicable DDLs. Replicable DDLs will be sent to the downstream by TiCDC.
+- `SECONDARY`: You cannot execute replicable DDLs or non-replicable DDLs. However, DDLs executed in a PRIMARY cluster can be replicated to a SECONDARY cluster by TiCDC.
 
-When no BDR role is set, you can execute any DDL. But after you set `bdr_mode=true` on TiCDC, the executed DDL will not be replicated by TiCDC.
+When not setting the BDR role, you can execute any DDL. However, the changefeed in BDR mode will not replicate any DDL on that cluster.
 
-> **Warning:**
->
-> This feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+In general, in BDR mode, TiCDC will only replicate replicable DDL on the Primary cluster to the downstream.
```
Comment (hfxsd) [inline ticdc/ticdc-bidirectional-replication.md:97]:
```suggestion
In general, in BDR mode, TiCDC will only replicate replicable DDL on the PRIMARY cluster to the downstream.
```

**Review by Frank945946 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-bidirectional-replication.md [hunk 2]
```diff
@@ -89,14 +89,12 @@ Non-replicable DDLs include:
 
 To solve the problem of replicable DDLs and non-replicable DDLs, TiDB introduces the following BDR roles:
 
-- `PRIMARY`: you can execute replicable DDLs, but cannot execute non-replicable DDLs. Replicable DDLs will be replicated to the downstream by TiCDC.
-- `SECONDARY`: you cannot execute replicable DDLs or non-replicable DDLs, but can execute the DDLs replicated by TiCDC.
+- `PRIMARY`: You can execute replicable DDLs, but not non-replicable DDLs. Replicable DDLs will be sent to the downstream by TiCDC.
```
Comment (lilin90) [inline ticdc/ticdc-bidirectional-replication.md:92]:
```suggestion
- `PRIMARY`: You can execute replicable DDLs, but not non-replicable DDLs. Replicable DDLs executed in a PRIMARY cluster will be replicated to the downstream by TiCDC.
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 3]
```diff
@@ -89,14 +89,12 @@ Non-replicable DDLs include:
...
+- `SECONDARY`: You cannot execute replicable DDLs or non-replicable DDLs. However, DDLs executed in a PRIMARY cluster can be replicated to a SECONDARY cluster by TiCDC.
 
-When no BDR role is set, you can execute any DDL. But after you set `bdr_mode=true` on TiCDC, the executed DDL will not be replicated by TiCDC.
+When not setting the BDR role, you can execute any DDL. However, the changefeed in BDR mode will not replicate any DDL on that cluster.
```
Comment (lilin90) [inline ticdc/ticdc-bidirectional-replication.md:95]:
```suggestion
When no BDR role is set, you can execute any DDL. However, the changefeed in BDR mode does not replicate any DDL on that cluster.
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 4]
```diff
@@ -89,14 +89,12 @@ Non-replicable DDLs include:
...
 
-> **Warning:**
->
-> This feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+In general, in BDR mode, TiCDC will only replicate replicable DDL on the PRIMARY cluster to the downstream.
```
Comment (lilin90) [inline ticdc/ticdc-bidirectional-replication.md:97]:
```suggestion
In short, in BDR mode, TiCDC only replicates replicable DDLs in the PRIMARY cluster to the downstream.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**